lopezMidterm/src/hooks/useFetch.tsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from 'react';
+import axios, { AxiosResponse } from 'axios';
+import React from 'react';
+
+interface FetchResult<T> {
+  data: T;
```

Comment:
Make sure you understand what `T` is, how it differs from the `any` type and why you would use it here instead of any`